axelrod/strategies/worse_and_worse.py

```diff
@@ -57,3 +57,72 @@ def strategy(self, opponent):
         expected_length = self.match_attributes['length']
         probability = 1 - float(current_round) / expected_length
         return random_choice(probability)
+
+
+class WorseAndWorse2(Player):
+    """
+
+    Plays as tit for tat during the first 20 moves.
+    Then defects with probability (current turn - 20) / current turn.
+    Therefore it is more and more likely to defect as the round goes on.
+
+    Names:
+        - Worse and Worse 2: [PRISON1998]_
+
+    """
+
+    name = 'Worse and Worse 2'
+    classifier = {
+        'memory_depth': float('inf'),
+        'stochastic': True,
+        'makes_use_of': set(),
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def strategy(self, opponent):
+        current_round = len(self.history) + 1
+
+        if current_round == 1:
+            return C
+        elif current_round <= 20:
+            return opponent.history[-1]
+        else:
+            probability = 20 / float(current_round)
+            return random_choice(probability)
```

Comment:
Do you think I should make a `random_choice_defects` function or something of the sort, as it might be a little confusing that the stated probability is `1 - 20 / current turn`, but the probability in the code is just `20 / current turn`?
Or should I add a comment to that effect?